### PR TITLE
Add k8s Pod/Cluster Prometheus Dashboards

### PR DESCRIPTION
**Pull request by algchoo**
* added dashboard json for k8s pod/cluster prometheus metrics
* included screenshots
* included readme

### CODE REVIEW

File: dashboards/k8s/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+sample_dashboards:
+  -
+    category: Kubernetes
+    id: kubernetes-cluster-overview
+    display_name: Kubernetes Cluster Overview
+    description: "This dashboard has charts displaying: Nodes (Ready), Nodes (Not Ready), Nodes Location, Scheduled Pods Per Cluster, Node CPU Usage, Node Memory Used, Persistent Volume Provisioned Capacity (Total), PV Bound, PV Unbound, PV Pending, PV Failed, and Persistent Volume Provisioned Capacity"
+    related_integrations:
+      - id: kubernetes
+        platform: GCE
```
Comment (EvanSimpson) [inline dashboards/k8s/metadata.yaml:9]:
```suggestion
        platform: GKE
```

File: dashboards/k8s/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  -
+    category: Kubernetes
+    id: kubernetes-pod-overview
+    display_name: Kubernetes Pod Overview
+    description: "This dashboard has charts displaying: Scheduled Pods, Running Pods, Pending Pods, Evicted Pods, Shutdown Pods, Failed Pods, Namespaces, Replicasets, Deployments, Daemonsets, Statefulsets, Jobs, Scheduled Pods Per Cluster, Failed Pods Per Cluster, Pod CPU Usage, Pod Memory Usage, Pod Network Receive Bytes, and Pod Network Transmit Bytes"
+    related_integrations:
+      - id: kubernetes
+        platform: GCE
```
Comment (EvanSimpson) [inline dashboards/k8s/metadata.yaml:17]:
```suggestion
        platform: GKE
```

File: dashboards/k8s/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    category: Kubernetes
+    id: kubernetes-cluster-overview
+    display_name: Kubernetes Cluster Overview
+    description: "This dashboard has charts displaying: Nodes (Ready), Nodes (Not Ready), Nodes Location, Scheduled Pods Per Cluster, Node CPU Usage, Node Memory Used, Persistent Volume Provisioned Capacity (Total), PV Bound, PV Unbound, PV Pending, PV Failed, and Persistent Volume Provisioned Capacity"
+    related_integrations:
+      - id: kubernetes
+        platform: GCE
+  -
+    category: Kubernetes
+    id: kubernetes-pod-overview
```
Comment (EvanSimpson) [inline dashboards/k8s/metadata.yaml:12]:
```suggestion
    id: k8s-pod-prometheus
```

File: dashboards/k8s/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+sample_dashboards:
+  -
+    category: Kubernetes
+    id: kubernetes-cluster-overview
```
Comment (EvanSimpson) [inline dashboards/k8s/metadata.yaml:4]:
```suggestion
    id: k8s-cluster-prometheus
```

File: dashboards/k8s/k8s-cluster-prometheus.json [hunk 5]
```diff
@@ -0,0 +1,318 @@
...
+      },
+      {
+        "height": 3,
+        "widget": {
+          "timeSeriesTable": {
+            "dataSets": [
+              {
+                "tableDisplayOptions": {},
+                "timeSeriesQuery": {
+                  "prometheusQuery": "sum by (cluster) (kube_node_status_condition{condition=\"Ready\",status=\"true\"})"
```
Comment (EvanSimpson) [inline dashboards/k8s/k8s-cluster-prometheus.json:264]:
```suggestion
                  "prometheusQuery": "sum by (cluster) (kube_node_status_condition{condition=\"Ready\",status=\"false\"})"
```

File: dashboards/k8s/k8s-cluster-prometheus.json [hunk 6]
```diff
@@ -0,0 +1,318 @@
...
+                "direction": "ABOVE",
+                "label": "",
+                "value": 0
+              },
+              {
+                "color": "YELLOW",
+                "direction": "ABOVE",
+                "label": "",
+                "value": 0
+              }
```
Comment (EvanSimpson) [inline dashboards/k8s/k8s-cluster-prometheus.json:164]: These have the same threshold, should there be different values, or can one be removed?

**Review by EvanSimpson — CHANGES_REQUESTED** [review]
Can you also update the folder name to be `kubernetes` instead of `k8s`?

**Comment by algchoo** [comment]
@EvanSimpson Thanks for the review! I've pushed changes for the feedback in regards to the IDs/filenames needing to match, as well as fixing any mistakes/issues with the dashboard json. I tested importing the json after making those changes and it worked in each case. Hope you're having a great day!

File: dashboards/kubernetes/metadata.yaml [hunk 7]
```diff
@@ -0,0 +1,17 @@
+sample_dashboards:
+  -
+    category: Kubernetes
+    id: k8s-cluster-prometheus
+    display_name: Kubernetes Cluster Overview
+    description: "This dashboard has charts displaying: Nodes (Ready), Nodes (Not Ready), Nodes Location, Scheduled Pods Per Cluster, Node CPU Usage, Node Memory Used, Persistent Volume Provisioned Capacity (Total), PV Bound, PV Unbound, PV Pending, PV Failed, and Persistent Volume Provisioned Capacity"
+    related_integrations:
+      - id: kubernetes
+        platform: GCE
```
Comment (EvanSimpson) [inline dashboards/kubernetes/metadata.yaml:9]:
```suggestion
        platform: GKE
```

File: dashboards/kubernetes/metadata.yaml [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+  -
+    category: Kubernetes
+    id: k8s-pod-prometheus
+    display_name: Kubernetes Pod Overview
+    description: "This dashboard has charts displaying: Scheduled Pods, Running Pods, Pending Pods, Evicted Pods, Shutdown Pods, Failed Pods, Namespaces, Replicasets, Deployments, Daemonsets, Statefulsets, Jobs, Scheduled Pods Per Cluster, Failed Pods Per Cluster, Pod CPU Usage, Pod Memory Usage, Pod Network Receive Bytes, and Pod Network Transmit Bytes"
+    related_integrations:
+      - id: kubernetes
+        platform: GCE
```
Comment (EvanSimpson) [inline dashboards/kubernetes/metadata.yaml:17]:
```suggestion
        platform: GKE
```

File: dashboards/kubernetes/k8s-cluster-prometheus.json [hunk 9]
```diff
@@ -0,0 +1,312 @@
...
+      },
+      {
+        "height": 3,
+        "widget": {
+          "timeSeriesTable": {
+            "dataSets": [
+              {
+                "tableDisplayOptions": {},
+                "timeSeriesQuery": {
+                  "prometheusQuery": "sum by (cluster) (kube_node_status_condition{condition=\"Ready\",status=\"true\"})"
```
Comment (EvanSimpson) [inline dashboards/kubernetes/k8s-cluster-prometheus.json:258]: Carrying this comment forward - should this have a status value of `false`?

**Review by EvanSimpson — CHANGES_REQUESTED** [review]
Thanks for all the changes! I've merged a number of the prometheus dashboards, and just have a couple of changes requested on the remaining open ones.

**Review by EvanSimpson — APPROVED** [review]

**Pull request merged**